src/main/java/net/dries007/tfc/api/capability/heat/CapabilityItemHeat.java

```diff
@@ -35,7 +35,7 @@ public static void preInit()
      */
     public static float adjustTemp(float temp, float heatCapacity, long ticksSinceUpdate)
     {
-        if (ticksSinceUpdate == -1) return MIN_TEMPERATURE;
```

Comment:
This line fixes an issue I noticed that sometimes, when there's a lag spike, the temp resets. For some reason, when that happens, this function gets ticksSinceUpdate param as -1. Also I think that it is worth mentioning that this issue happens not only for ingots but clay casts as well inside the charcoal forge, so, in any inventory slot with any temperature capability item.

Reply:
It seems like it's a desync error with the calendar. If that's the case, I can change the "default value" of `lastUpdateTick` to `Integer.MIN_VALUE`. That should avoid this issue. I'll have to test this though, because I'm pretty sure replacing that line will cause stackability issues.

Reply:
Scratch that, I poked around in code and I'm misunderstanding the issue. This is fine as is.